topydo/commands/PriorityCommand.py

```diff
@@ -36,7 +36,7 @@ def normalize_priority(p_priority):
             match = re.search(r'\b([A-Z])\b', p_priority)
             return match.group(1) if match else p_priority
 
-        priority = normalize_priority(self.args[-1])
+        priority = normalize_priority(self.args[-1].upper())
```

Comment:
I see it as a part of normalization, so I'd expect `upper()` to be called in the `normalize_priority` function.

```python
def normalize_priority(p_priority):
    match = re.search(r'\b([A-Z])\b', p_priority.upper())
    return match.group(1) if match else p_priority
```